dom-events.tar/dom-events/buttons.js

```diff
@@ -22,6 +22,6 @@ function handleDoubleClick(event){
   console.log(event);
   console.log(event.target);
 }
- const element=document.querySelector(.'double-click-button');
+ const element3=document.querySelector(.'double-click-button');
   if (!$dblClickButton) throw new Error('The $dblClickButton query failed');
   element.addEventListener('dblclick', handleDoubleClick);
```

Comment:
same here, make sure you're adding the event listener on the right element (`element3` in this case)